### ISSUE TITLE
How to change i3 settings

### ISSUE THREAD

**Issue by gmoneymagna** [unix121/i3wm-themer#70]
Hello noob here. I'm wondering if I can change my i3 settings while using this theme selector. It looks great but my i3 config changes about workspace positions and applications is gone. That is fine I can re-input them but I'm not sure how to do that. Modifying my original ~/.i3/config file doesn't change anything. I assume now my system is looking elsewhere for these settings?

Love the look of these themes, I'm just hoping to be able to configure it appropriately.

Cheers.

**Comment by unix121** [comment]
Hello sir! Yeah it is still possible, if you are editing the correct `i3 config` file then on system refresh it should display the changes automatically just like before.

Are you sure you are editing the one located in `/$HOME/.i3/config` (or) `/$HOME/.config/i3/config` ?

Generally the script doesn't override the location of your i3 configuration file so there is probably something else going on.

If the problem persists then maybe post a bit more information to help me figure it out ;)

**Comment by gmoneymagna** [comment]
Thank you for your quick response.

I have always kept my config in home/$USR/.i3/config (this is how i3 manjaro installed). I thought I found the problem when in the config.yaml file i3 config wasn't pointing to the correct path. However, when I changed the path to my .i3/config location, it hasn't fixed the problem.

**Comment by gmoneymagna** [comment]
Thank you for your quick response.

I have always kept my config in home/$USR/.i3/config (this is how i3 manjaro installed). I thought I found the problem when in the config.yaml file i3 config wasn't pointing to the correct path. However, when I changed the path to my .i3/config location, it hasn't fixed the problem.

**Comment by unix121** [comment]
Did you run the installation for the defaults once more after changing the path inside `config.yaml` ?

`python i3wm-themer.py --config config.yaml --install defaults/`

So that it copies the defaults once more.

(I have run the script multiple times in Manjaro i3 once it was my main distro for ages and never run into a problem so far)

**Comment by gmoneymagna** [comment]
okay I have reinstalled the config defaults script (I didn't do that before) and reinstalled the theme I want. The config.yaml file points in the correct path. However when I modify that file, nothing changes.

I will continue to look. Honestly I would be happy just knowing which i3config file to modify so I can transfer over my settings.

**Comment by gmoneymagna** [comment]
okay when I modify ~/.config/i3/config it alters the settings. So it appears I can modify this file with my old settings. Just a little confused why the script is reading that file instead of the file the config.yaml is pointing to.

**Comment by gmoneymagna** [comment]
Now my problem is that when I modify ~/.config/polybar/config (to display the bar on my primary monitor) the changes do not persist after a login even though the content remains modified.

Also, when I change the names of my workspaces in the ~/.config/i3/config file they show up in polybar as "-1". However, this may be strictly a polybar issue so I may look elsewhere for the answer.

**Comment by unix121** [comment]
Yeah, that is kinda odd but I will try to reproduce it in a WM later maybe. I wish I could help you like this, but troubleshooting totally blind is kinda hard :/

If you happen to find the solution please post it here so that anyone who might meet this issue can find a solution :)

**Comment by gmoneymagna** [comment]
Thank you for your help!

I have solved the polybar issue by replacing %index% with %name% in the polybar config file. If you place a number before the name in the i3config file (ex: "7:Chat) then polybar still keeps the workspaces in numerical order.

Am I correct in understanding that none of these changes I have made to i3config and polybarconfig will revert to default if I install a new theme?
